packages/ui/csg-viewer/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@jscad/csg-viewer",
+  "version": "0.0.0",
+  "description": "",
+  "main": "src/index.js",
+  "scripts": {
+    "test": "tape test.js",
+    "dev": "budo example/demo.js --title csg-viewer --live --css example/demo.css -- -t glslify -t brfs",
+    "build": "browserify -p common-shakeify --standalone makeCsgViewer src/index.js > dist/bundle.js",
+    "minify": "uglifyjs dist/bundle.js --compress > dist/bundle.min.js",
+    "compress": "gzipme dist/bundle.min.js",
+    "build-min": "npm run build && npm run minify && npm run compress"
+  },
+  "author": "Mark 'kaosat-dev' Moissette",
+  "license": "MIT",
+  "devDependencies": {
+    "@jscad/scad-api": "^0.4.2",
```

Comment:
Why the dependency on @jscad/scad-api?

Reply:
devDependency: ie for tests/ demo to illustrate how you can use to view your csg stuff
we can remove that in the course of V2